release_notes/gitops-release-notes-1-19.adoc

```diff
@@ -0,0 +1,45 @@
+//OpenShift GitOps Release Notes
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="gitops-release-notes"]
+= {gitops-title} release notes
+:context: gitops-release-notes
+
+toc::[]
+
+Release notes contain information about new and deprecated features, breaking changes, fixed issues, and known issues. The following release notes apply to the most recent OpenShift {gitops-shortname} releases on {OCP}.
+
+{gitops-title} is a declarative way to implement continuous deployment for cloud native applications. {gitops-title} ensures consistency in applications when you deploy them to different clusters in different environments, such as development, staging, and production. {gitops-title} helps you automate the following tasks:
+
+* Ensure that the clusters have similar states for configuration, monitoring, and storage.
+* Recover or recreate clusters from a known state.
+* Apply or revert configuration changes to multiple {OCP} clusters.
+* Associate templated configuration with different environments.
+* Promote applications across clusters, from staging to production.
+
+For an overview of {gitops-title}, see xref:../understanding_openshift_gitops/about-redhat-openshift-gitops.adoc#about-redhat-openshift-gitops[About {gitops-title}].
+
+[NOTE]
+====
+For additional information about the OpenShift {gitops-shortname} lifecycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red{nbsp}Hat {OCP} Life Cycle Policy].
+====
+
+// Compatibility and support matrix
+include::modules/go-compatibility-and-support-matrix.adoc[leveloffset=+1]
+
+// Modules included, most to least recent
+
+// Release notes for Red Hat OpenShift GitOps 1.19.0
+include::modules/gitops-release-notes-1-19-0.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
+
+* xref:../argo_cd_agent_architecture/argocd-agent-architecture-overview.adoc[Argo CD Agent architecture overview]
+
+* xref:../argocd_agent_installation/argocd-agent-installation.adoc[Installing Argo CD Agent]
+
+* xref:../argocd_instance/setting-up-argocd-instance.adoc#gitops-configuring-imagepullpolicy[Configuring image pull policy]
```

Comment:
This is not redirecting me to the main content. 

This link redirect to correct content
https://103017--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance#gitops-configuring-imagepullpolicy_setting-up-argocd-instance 

Can you confirm again?

Reply:
This comment is fixed now @svghadi, please check.